bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/FormBrowser.java

```diff
@@ -36,15 +46,42 @@ public void createControl(Composite parent) {
 		flayout.marginWidth = 1;
 		flayout.marginHeight = 1;
 		container.setLayout(flayout);
-		formText = new ScrolledFormText(container, SWT.V_SCROLL | SWT.H_SCROLL, false);
+		formText = new Browser(container, SWT.NONE);
+		LocationListener locationListener = new LocationListener()
+		{
+
+			@Override
+			public void changing(LocationEvent event)
+			{
+				try
+					{
+						PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser()
+								.openURL(new URL(event.location));
+						event.doit = false;
+					}
+					catch (
+							PartInitException
+							| MalformedURLException e)
+					{
+						e.printStackTrace();
```

Comment:
Log the exception

Reply:
@kolipakakondal 
I used  IDF Eclipse plug-in logger:
https://github.com/espressif/idf-eclipse-plugin/blob/db760d63e4174b294d5c8753cebe7ffced8ccdc1/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/FormBrowser.java#L62-L67